solara/lab/components/chat.py

```diff
@@ -32,7 +32,8 @@ def ChatBox(
     if "overflow-y" not in style_flat:
         style_flat += " overflow-y: auto;"
 
-    classes += ["chat-box"]
+    if "chat-box" not in classes:
+        classes += ["chat-box"]
```

Comment:
I think the problem was mutating it, so I think this would be a cleaner fix. What do you think?
```suggestion
    classes = [*classes, "chat-box"]
```